File: EGIS_Management/Domain/Entities/toims/CheckReceivedDay.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities.toims
{
    public class CheckReceivedDay
    {
        [Key] public int Id { get; set; }
        public int ReceiptNumber { get; set; }
        public int BankNumber { get; set; }
        public string BankName { get; set; }
        public int CheckNumber { get; set; }
        public double CheckAmount { get; set; }
        public DateOnly CheckDate { get; set; }
        public DateTime CreateAt { get; set; }
    }
}

File: SPA19/EGIS/src/app/toims/models/checkreceivedday.ts
export interface ICheckReceivedDay {
    id: number;
    receiptNumber: number;
    bankNumber : number;
    bankName : string; 
    checkNumber : number;
    checkAmount : number;
    checkDate : Date;
 }

 export type createCheckReceivedDayDto =Omit<ICheckReceivedDay,'receiptNumber'|'bankNumber'|'bankName'|'checkNumber'|'checkAmount'|'checkDate'>;